File: Lesson 36/Lesson 35/FirstViewController.m
//
//  FirstViewController.m
//  Lesson 35
//
//  Created by Lucas Derraugh on 4/2/12.
//  Copyright (c) 2012 __MyCompanyName__. All rights reserved.
//

#import "FirstViewController.h"

@interface FirstViewController ()

@end

@implementation FirstViewController
@synthesize textField = _textField;

- (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
{
    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
    if (self) {
        // Initialization code here.
    }
    
    return self;
}

- (IBAction)changeText:(id)sender {
    [_textField setStringValue:@"Changed Value"];
}


@end

File: Lesson 81/Lesson 81/FirstViewController.swift
//
//  FirstViewController.swift
//  Lesson 81
//
//  Created by Lucas Derraugh on 4/3/20.
//  Copyright © 2020 Lucas Derraugh. All rights reserved.
//

import Cocoa

class FirstViewController: NSViewController {
    
    override init(nibName nibNameOrNil: NSNib.Name?, bundle nibBundleOrNil: Bundle?) {
        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
        title = "First"
    }
    
    required init?(coder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
}
